.github/workflows/master-build.yaml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
     - main
+    - v1.2.0
+    - image-name
```

Comment:
nit: when PR is done, should remove this line probably?

Reply:
Good point. Let me remove the `image-name` already.